### PR TITLE
SS-37

### PR DESCRIPTION
**Pull request by wongck0426**
Added basic front end components for product listing in "mystore".
Added a icon for my store on right side of the header

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +0,0 @@
-FROM openjdk:19-jdk-alpine
```
Comment (nels-dev) [inline Dockerfile:1]: why delete Dockerfile 😨

File: frontend/src/components/ListingTable.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      <v-table class="ma-15" style="margin: 0 auto; width:65%" :data="listings">
+        <thead>
+          <tr>
+            <th class="text-left" @click="sortListing('name')">
+              Product name
+            </th>
+            <th class="text-left">
+              Description
+            </th>
+            <th class="text-left" @click="sortListing('price')">
```
Comment (nels-dev) [inline frontend/src/components/ListingTable.vue:13]: where is sortListing defined?

File: frontend/src/components/ListingTable.vue [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            <th class="text-left">
+              Status
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="item in listings" :key="item.name">
+            <td>{{ item.name }}</td>
+            <td>{{ item.description }}</td>
+            <td>${{ item.price }}</td>
```
Comment (nels-dev) [inline frontend/src/components/ListingTable.vue:28]: better align with the dollar sign ('C') in product listing page

**Comment by nels-dev** [comment]
please rebase main

File: frontend/src/components/ListingTable.vue (hunk 3 above)
Comment (wongck0426) [reply frontend/src/components/ListingTable.vue:28]: use C$

**Review by nels-dev — APPROVED** [review]

**Pull request merged**